spec/models/validations/sales/financial_validations_spec.rb

```diff
@@ -266,4 +266,86 @@
       end
     end
   end
+
+  describe "#validate_equity_in_range_for_year_and_type" do
+    let(:record) { FactoryBot.create(:sales_log, saledate: now) }
+
+    before do
+      Timecop.freeze(now)
+    end
+
+    after do
+      Timecop.unfreeze
+    end
```

Comment:
the before and after can be swapped for an around
you can search the syntax in the project but something like 
```
before do |test|
Timecop.freeze
test.run
Timecop.unfreeze
end
```

Reply:
yeah true, in case you're interested an around actually behaves slightly differently to a before/after in terms of when it is called but in this case you're right they're interchangeable